streams/za.m3u

```diff
@@ -30,28 +30,22 @@ https://webstreaming-8.viewmedia.tv/web_119/Stream/playlist.m3u8
 #EXTINF:-1 tvg-id="ROVTV.za",ROV TV (576p)
 https://webstreaming.viewmedia.tv/web_012/Stream/playlist.m3u8
 #EXTINF:-1 tvg-id="SABC1.za",SABC 1 (720p)
-http://185.59.221.131:8081/live/sabc1/playlist.m3u8
+https://sabconeta.cdn.mangomolo.com/sabc1/smil:sabc1.stream.smil/chunklist_b1600000_t64NzIwcA==.m3u8
```

Comment:
```suggestion
https://sabconeta.cdn.mangomolo.com/sabc1/smil:sabc1.stream.smil/master.m3u8
```

Adaptive.

![image](https://github.com/user-attachments/assets/d8dc5103-fd2e-46d5-b32b-a9935c898889)

(tested via VPN in Johannesburg)